From 1b0d134e382cc03c0d24c6fce332fe02a17305de Mon Sep 17 00:00:00 2001
From: Mark D Horn <mark.d.horn@intel.com>
Date: Thu, 17 Dec 2020 10:35:31 -0800
Subject: [PATCH] Fix isAlive -> is_alive for python3.9

The isAlive() method of threading.Thread has been removed.
It was deprecated since Python 3.8. Use is_alive() instead.

Signed-off-by: Mark D Horn <mark.d.horn@intel.com>
---
 util/kojira | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/util/kojira b/util/kojira
index a9e5fc99bcb4..63ed22529968 100755
--- a/util/kojira
+++ b/util/kojira
@@ -928,13 +928,13 @@ def main(options, session):
             repomgr.checkQueue()
             repomgr.printState()
             repomgr.pruneLocalRepos()
-            if not curr_chk_thread.isAlive():
+            if not curr_chk_thread.is_alive():
                 logger.error("Currency checker thread died. Restarting it.")
                 curr_chk_thread = start_currency_checker(session, repomgr)
-            if options.check_external_repos and not curr_ext_chk_thread.isAlive():
+            if options.check_external_repos and not curr_ext_chk_thread.is_alive():
                 logger.error("External currency checker thread died. Restarting it.")
                 curr_ext_chk_thread = start_external_currency_checker(session, repomgr)
-            if not regen_thread.isAlive():
+            if not regen_thread.is_alive():
                 logger.error("Regeneration thread died. Restarting it.")
                 regen_thread = start_regen_loop(session, repomgr)
         except KeyboardInterrupt:
-- 
2.29.2

